monitoring/configuring-platform-monitoring.adoc

```diff
@@ -1,5 +1,5 @@
-[id="configuring-the-monitoring-stack"]
-= Configuring the monitoring stack
+[id="configuring-platform-monitoring"]
+= Configuring platform monitoring
 include::modules/common-attributes.adoc[]
 :context: configuring-monitoring
```

Comment:
Make sure the `id` and `:context:` match

Reply:
Resolved for all monitoring assemblies.